### PR TITLE
create a recommendation file for blore

### PR DESCRIPTION
**Pull request by SanjayYashawant**
Things to Do:

- [ ] Add commit
- [ ] Review comment
- [ ] Official review
- [ ] Merge

### CODE REVIEW

**Comment by emcjoe** [comment]
This is a general comment

File: sanjaysy456-bangalore.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Bangalore , Karnataka
+
+### places to eat
+- city taj
+- subway
+
+### Paces to  visualizing-git
```
Comment (emcjoe) [inline sanjaysy456-bangalore.md:7]: Places to visit?

File: sanjaysy456-bangalore.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+-palace
+-zoo
+
+#### places to avoid
+-hills
+-park
```
Comment (kumardss) [inline sanjaysy456-bangalore.md]: :rofl:

**Review by emcjoe — CHANGES_REQUESTED** [review]
Think you need to address the comment - either adjust text or tell me to go away :-)

**Review by elena631905 — APPROVED** [review]

**Pull request merged**